pkg/controller/upload_controller.go

```diff
@@ -197,7 +197,7 @@ func (c *uploadController) pvbHandler(obj interface{}) {
 		return
 	}
 
-	log.Debug("Enqueueing Upload")
+	log.Info("Enqueueing Upload")
```

Comment:
What object are we enqueueing the upload for?

Reply:
The upload CR key will also be logged due the previous configuration of log fields in the same function.

`log := loggerForUpload(c.logger, req)`